tests/integration_tests.rs

```diff
@@ -966,7 +966,7 @@ fn filename_stdin_binary() {
     let vec = vec![0; 1];
     bat_with_config()
         .arg("--decorations=always")
-        .arg("--style=header")
+        .arg("--style=header-filename")
```

Comment:
In general I would prefer if we kept existing regression tests unchanged, because that is the safest way to make sure we don't have any regressions.

Sometimes it makes sense to change integration tests, but e.g. changing `header` to `header-filename` everywhere is not something we should do IMHO.

So I would prefer if you added new tests for the new style components.

I want to stress again: Please use your good judgement to decide when to side-step that general guideline.

Reply:
I tried to avoid these renames as muchas possible and in cases where testing the header was the goal (and not just filename), I copied the tests over and added `header-filename,header-filesize` to test those two together.